testing/test_events.py

```diff
@@ -0,0 +1,57 @@
+import pytest
+import eth_utils
+from ethindex import pgimport
+
+
+def make_address(i: int):
+    return eth_utils.to_checksum_address("0x{:040X}".format(i))
+
+
+@pytest.fixture
+def add_some_tranfer_events(testenv):
```

Comment:
This is a weird way of using this for me, as you only use the side effect, and later in the tests it seems unused. 
I would rather name it `testenv_with_transfers` and return `testenv` , so then you can later only use `testenv_with_transfers` as the fixture.

Reply:
I don't think this much better. Can we postpone the decision on this? I'd rather look into a way of using eth_tester snapshots.

Reply:
ok, I see your point. I would still like to explore eth_tester snapshots and change this later.

Reply:
Then please create an issue